nms/app/packages/magmalte/app/views/subscriber/SubscriberDetailConfig.js

```diff
@@ -122,24 +122,82 @@ export default function SubscriberDetailConfig() {
           </Grid>
           <Grid container spacing={4}>
             <Grid item xs={12} md={6}>
-              <CardTitleRow label="Subscriber" filter={EditSubscriberButton} />
+              <CardTitleRow
+                label="Subscriber"
+                filter={EditSubscriberInfoButton}
+              />
               <SubscriberInfoConfig />
             </Grid>
 
             <Grid item xs={12} md={6}>
               <CardTitleRow
                 label="Traffic Policy"
-                filter={EditSubscriberButton}
+                filter={EditSubscriberTrafficPolicyButton}
               />
               <SubscriberConfigTrafficPolicy />
             </Grid>
+            <Grid item xs={12} md={6}>
+              <CardTitleRow
+                label="APN Static IPs"
+                filter={EditSubscriberApnStaticIpsButton}
+              />
+              <SubscriberApnStaticIps />
+            </Grid>
           </Grid>
         </Grid>
       </Grid>
     </div>
   );
 }
 
+function EditSubscriberInfoButton() {
+  const [open, setOpen] = useState(false);
+  return (
+    <>
+      <SubscriberEditDialog
+        open={open}
+        onClose={() => setOpen(false)}
+        editProps={{editTable: 'subscriber'}}
+      />
+      <Button component="button" variant="text" onClick={() => setOpen(true)}>
+        {'Edit'}
+      </Button>
+    </>
+  );
+}
+
+function EditSubscriberTrafficPolicyButton() {
+  const [open, setOpen] = useState(false);
+  return (
+    <>
+      <SubscriberEditDialog
+        open={open}
+        onClose={() => setOpen(false)}
+        editProps={{editTable: 'trafficPolicy'}}
+      />
+      <Button component="button" variant="text" onClick={() => setOpen(true)}>
+        {'Edit'}
+      </Button>
+    </>
+  );
+}
+
+function EditSubscriberApnStaticIpsButton() {
+  const [open, setOpen] = useState(false);
+  return (
+    <>
+      <SubscriberEditDialog
+        open={open}
+        onClose={() => setOpen(false)}
+        editProps={{editTable: 'staticIps'}}
+      />
+      <Button component="button" variant="text" onClick={() => setOpen(true)}>
+        {'Edit'}
+      </Button>
+    </>
+  );
```

Comment:
I think that here it's better to increase code re-use by combining these three into a single component with an enum prop field to control which tab the dialog opens up with